pontoon/sync/vcs/models.py

```diff
@@ -527,9 +582,10 @@ def l10n_base(self):
     @cached_property
     def parsed_configuration(self):
         """Return parsed project configuration file."""
-        return TOMLParser().parse(
-            self.configuration_path, env={"l10n_base": self.l10n_base},
```

Comment:
That means we can also remove other traces of `configuration_path` from this file and the test file.